sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 #### Bugs Fixed
 
 #### Other Changes
+* Remove requires io.netty.transport.epoll from module-info.
```

Comment:
please remove `Remove requires io.netty.transport.epoll from module-info.`. - this change has already been released

Reply:
done